main.go

```diff
@@ -88,6 +88,23 @@ func main() {
 		},
 	)
 
+	if vargs.UpdateEnvironment == true {
```

Comment:
I missed the comment. We need an err check before the environment because it should already bail out there if the step before failed.